sympy/physics/vector/dyadic.py

```diff
@@ -55,6 +56,12 @@ def __init__(self, inlist):
                 i -= 1
             i += 1
 
+    @property
+    def func(self):
+        return Dyadic
+
+    is_number = False
```

Comment:
Can you move the class attributes above all the methods? It is easier to find if they are organized that way.

Reply:
Yes, sure.